### PR TITLE
API: Add documentation about rate limiting

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

**Comment by nicpottier** [comment]
Shouldn't we say what the rate limit is?

**Comment by ycleptkellan** [comment]
Yeah, hard to plan without knowing the limit. Would be helpful for larger orgs.

**Comment by rowanseymour** [comment]
Have added the limit, but made it clear that there's no guarantee it won't change in future. That seem reasonable?

File: temba/api/v2/views.py [hunk 1]
```diff
@@ -79,14 +79,24 @@ class RootView(views.APIView):
...
-     * **403**: You do not have permission to access this resource
-     * **404**: The resource was not found (returned by `POST` and `DELETE` methods)
+     * **403**: You do not have permission to access this resource.
+     * **404**: The resource was not found (returned by `POST` and `DELETE` methods).
+     * **429**: You have exceeded the rate limit for this endpoint (see below).
+
+    ## Rate Limiting
+
+    All endpoints are subject to rate limiting. If you exceed the number of allowed requests in a given time window, you
+    will get response with status code 429. The response will also include a header called 'Retry-After' which will
```
Comment (ericnewcomer) [inline temba/api/v2/views.py]: will get _a_ response

File: temba/api/v2/views.py [hunk 2]
```diff
@@ -79,14 +79,24 @@ class RootView(views.APIView):
...
+    specify the number of seconds that you should wait for before making further requests.
+
+    Currently the limit for all endpoints is 2500 requests per hour, but this may change in future so it is important
+    to check for 429 responses.
```
Comment (ericnewcomer) [inline temba/api/v2/views.py]: The rate limit for all endpoints is 2,500 requests per hour. It is important to honor the Retry-After header when encountering 429 responses as the limit is subject to change without notice.

Comment (rowanseymour) [inline temba/api/v2/views.py]: Updated

**Pull request merged**